leveler/leveler.py

```diff
@@ -2381,6 +2381,135 @@ async def disp_backgrounds(self, ctx, bg_type: str):
+
+                if r.status == 200:
+                    data = await r.json()
+                else:
+                    return await ctx.send("No data was found within the Mee6 API.")
+
+            for userdata in data["players"]:
+                await asyncio.sleep(0)
+                # _handle_levelup requires a Member
+                user = ctx.guild.get_member(int(userdata["id"]))
+
+                if not user:
+                    failed += 1
+                    continue
+
+                level = userdata["level"]
+                server = ctx.guild
+                channel = ctx.channel
+
+                # creates user if doesn't exist
+                await self._create_user(user, server)
+                userinfo = await self.db.users.find_one({"user_id": str(user.id)})
+
+                # get rid of old level exp
+                old_server_exp = 0
+                for _i in range(userinfo["servers"][str(server.id)]["level"]):
+                    await asyncio.sleep(0)
+                    old_server_exp += self._required_exp(_i)
+                userinfo["total_exp"] -= old_server_exp
+                userinfo["total_exp"] -= userinfo["servers"][str(server.id)]["current_exp"]
+
+                # add in new exp
+                total_exp = self._level_exp(level)
+                userinfo["servers"][str(server.id)]["current_exp"] = 0
+                userinfo["servers"][str(server.id)]["level"] = level
+                userinfo["total_exp"] += total_exp
+
+                await self.db.users.update_one(
+                    {"user_id": str(user.id)},
+                    {
+                        "$set": {
+                            "servers.{}.level".format(server.id): level,
+                            "servers.{}.current_exp".format(server.id): 0,
+                            "total_exp": userinfo["total_exp"],
+                        }
+                    },
+                )
+                await self._handle_levelup(user, userinfo, server, channel)
+        await ctx.send(f"{failed} users could not be found and were skipped.")
+
+    @checks.is_owner()
+    @mee6.command(name="roles", aliases=["ranks"])
+    @commands.guild_only()
+    async def convertranks(self, ctx):
+        """Convert Mee6 role rewards.
+        This command must be run in a channel in the guild to be converted."""
+        async with self.session.get(
+            f"https://mee6.xyz/api/plugins/levels/leaderboard/{ctx.guild.id}"
+        ) as r:
+            if r.status == 200:
+                data = await r.json()
+            else:
+                return await ctx.send("No data was found within the Mee6 API.")
+        server = ctx.guild
+        remove_role = None
+        for role in data["role_rewards"]:
+            await asyncio.sleep(0)
+            role_name = role["role"]["name"]
+            level = role["rank"]
+
+            role_obj = discord.utils.find(lambda rol: rol.name == role_name, server.roles)
```

Comment:
```suggestion
            role_obj = discord.utils.get(server.roles, name = role_name)
```
*just to silence codefactor*